minio/console
# Enable console to set expired-object-all-versions

This PR enables console to set bucket ILM rule for deleting all versions of expired object.

Steps to test:
Create a bucket and enable versioning for the same.
Try to create ILM rule and select option for delete all versions.
Make sure listing of rules works fine.
